README.md

```diff
@@ -32,7 +32,13 @@
 ### Launch source{d} lookout alpha at GopherCon
 
 - [ ] Release a production ready analyzer SDK
+  * Any developer able to understand how to create a new analizer
+  * Testing analyzer locally is easy: streamlined to 1-2 shell commands
 - [ ] Release a production ready service
+  * source{d} hosted deployment of lookout service
+  * Private 'status page' of the service health
+  * Automated CI/CD pipeline for staging/production environments
+  * Any OSS project (on Github) able to add lookout service (through 'Github Apps' flow)
```

Comment:
This means that the service should be approved by GitHub, and I don't know the workflow or requirements for this.

Reply:
Does it really need to be approved? https://developer.github.com/apps/building-github-apps/creating-a-github-app/

Reply:
AFAIK it does not need to be approved, and anyone can create one in Developer Settings, ofcourse untill we want to sell it on "GitHub Marketplace", wich is not the case at this Q. 

I also found this link helpful in understanding our options of integrating \w GitHub https://developer.github.com/apps/differences-between-apps/